_docs/concepts/what-is-grafeas/overview.md

```diff
@@ -1,223 +1,108 @@
-## Definition of terms
-**Notes**: A note is an item or condition that can be found via an analysis or something that is used multiple times in a process. For example, a CVE could be the result of a vulnerability analysis of a Linux package. In a build process, we would store information about our builder in a note. 
-
-A note name should take the format `/projects/<project_id>/notes/<note_id>` where the project_id would typically be different from the project where the occurrence is created and the note_id would be unique per note-project, and informative if possible. 
+# Grafeas: A Component Metadata API
 
-Access to notes should be read-only for users who have access to occurrences referencing them, and editable only by the note owner.
+Grafeas defines an API spec for managing metadata about software artifacts, such
+as VM images, jar files, scripts, and so on. You can use Grafeas to define and
+aggregate information about your project's components.
 
-**Occurrences**: An occurrence can be thought of as an instantiation of a note and describes how the note was found in a specific cloud resource or project (e.g., location, specific remediation steps, etc.), or what the results of a specific note were (e.g., the container images that resulted from a build). For example, an occurrence might report that the heartbleed OpenSSL bug (a possible Note) was found in a specific package of a container image, and include information about how to remedy the heartbleed bug based on the customer’s package.
+Grafeas divides metadata information into [_notes_](#notes), which contain
+high-level information, and [_occurrences_](#occurrences), which contain
+information about how and when notes occur in a particular project. This
+division allows third party metadata providers to create and manage metadata on
+behalf of many customers. It also allows for fine-grained access control of
+different types of metadata.
 
-An occurrence name should take the format `/projects/<project_id>/occurrences/<occurrence_id>` where the project_id would typically be different from the project where the note is created and the occurrence_id would be unique per occurrence-project, and would often be random. 
+## Definition of terms
 
-Write access to occurrences should only be granted to users who have access to link a note to the occurrence. Any users can have read access to occurrences. 
+### Notes
+
+A _note_ describes a high-level piece of metadata. Notes are generally found via
+analysis, and are things that occur multiple times across different projects.
+For example, you could create a note about a particular vulnerability after
+analyzing a Linux package. You would also use a note to store information about
+the builder of a build process. Notes are often owned and created by providers
+who analyze users' packages.
+
+Note names should follow the format `/projects/<project_id>/notes/<note_id>`.
+The note id needs to be unique per note-project, and informative if possible.
```

Comment:
@vtsao do you have thoughts on what this might look like? Not having actually seen notes "in the wild" I don't have a good example off the top of my head.